### DISCUSSION TITLE
Nordigen remember multiple accounts?

### DISCUSSION THREAD

**Discussion by leedare** [firefly-iii/firefly-iii#8002] [Firefly III]
The documentation around Nordigen isn't clear. Should I be able to run an Nordigen import that imports from 4 separate banks in 1 go?

If connect my account in the Nordigen portal, these are not seen in Firefly and I have to go through the authorise process every time.

If I run a Nordigen import from firefly it wants to connect to 1 bank only, and goes through the authentication process every time, so I do not see how I can automate the import?

If I save the config, and upload the config file, it will load the defaults for that bank only and go through the process once again.

So can I not run an import that pulls the data that Nordigen has collected already?

**Comment by JC5** [comment]
This can be done, if you create separate configuration files. There's only room for one bank in a configuration file. But you can use multiple configuration files with no trouble.

**Reply by leedare** [reply]
How can I select multiple configuration files during the import? It only lets me select one.

**Reply by JC5** [reply]
Yes I know. One at a time.
